### PR TITLE
Add build recipe for local CCDB application

### PR DESCRIPTION
**Pull request by mfasDa**

### CODE REVIEW

File: localccdb.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package: localccdb
+version: "%(tag_basename)s"
+tag: "current"
+requires:
+  - JDK
+---
+#!/bin/bash -e
+
+URL="http://alimonitor.cern.ch/download/local.jar"
```
Comment (ktf) [inline localccdb.sh:9]: Can we at least have the version in the URL, so that we automatically refetch it when the version change in the recipe?

Comment (costing) [reply localccdb.sh:9]: The current version is http://alimonitor.cern.ch/download/local-1.0.5.jar . +1 to use this URL instead.

Comment (mfasDa) [reply localccdb.sh]: Of course I am also more in favour of using a versioned package. It is fixed now. Also the tag is now specified to v1.0.5

**Comment by mfasDa** [comment]
If the PR is OK could you please merge it? I having the local CCDB as part of alidist could be helpful for other developers as well which are working with local test setups.

**Pull request merged**